packages/account/src/Components/personal-details/__tests__/personal-details.spec.js

```diff
@@ -42,8 +42,6 @@ const mock_errors = {
 const fake_alert_messaget =
     /we need this for verification\. if the information you provide is fake or inaccurate, you won’t be able to deposit and withdraw\./i;
 
-const checkbox_text =
-    /i hereby confirm that the tax information i provided is true and complete\. i will also inform deriv investments \(europe\) limited about any changes to this information\./i;
```

Comment:
I'm removing these string based tests. We should replace them with better ones, but I don't want to do it in this PR since it's few dozens test cases affected.